lab_1/main.py

```diff
@@ -62,7 +212,29 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict) or not isinstance(top_n, int):
+        return None
+
+    unk = unknown_profile['freq']
+    lan = profile_to_compare['freq']
+
+    comp = []
+    for i in unk:
+        for word in lan:
+            if i == word:
+                comp.append(i)
+
+    score = round(len(comp)/top_n, 1)
+    return score
+
+top_n_en = len(get_top_n_words(freq_dict_en, top_n_en))
+top_n_de = len(get_top_n_words(freq_dict_de, top_n_de))
+top_n_unknown = len(get_top_n_words(freq_dict_unknown, top_n_unknown))
+
+en_profile = create_language_profile(language_en, en_text, en_stop_words)
+de_profile = create_language_profile(language_de, de_text, de_stop_words)
+unknown_profile = create_language_profile(language_unknown, unknown_text, unknown_stop_words)
```

Comment:
The `main.py` file should not contain function calls - you should check your implementation using the `start.py` file.